share/smack/include/pthread.h

```diff
@@ -18,73 +20,95 @@
 #define PTHREAD_MUTEX_INITIALIZER                                              \
   { UNLOCKED, INITIALIZED }
 
-// Declare corral primitive procedures
-//__SMACK_INIT(corral_primitives);
-
-// Initialize thread status tracking variables (OS level behavior)
-//__SMACK_INIT(thread);
-
+// Helper functions
 void __VERIFIER_atomic_begin();
```

Comment:
While we’re cleaning this up, can we move `__VERIFIER_atomic_*` to a more-appropriate place, e.g., `smack.h`?

Reply:
Done.